topics/proteomics/metadata.yaml

```diff
@@ -123,13 +123,36 @@ material:
       - "Protein identification from LC-MS/MS raw files."
     time_estimation: "45min"
     key_points:
-      - "LC-MS/MS raw files have to be converted before using GalaxyP on most GalaxyP servers."
+      - "LC-MS/MS raw files have to be converted to mgf/mzML before using GalaxyP on most GalaxyP servers."
```

Comment:
The specific mentioning of GalaxyP without explanation might be a bit confusing? What is a "GalaxyP" server, what is "using GalaxyP"? It's all Galaxy after all. Would just say: "LC-MS/MS raw files have to be converted to mgf/mzML for further analysis."